Sources/Atomic.swift

```diff
@@ -107,80 +107,114 @@ internal struct UnsafeAtomicState<State: RawRepresentable> where State.RawValue
 /// `Lock` exposes `os_unfair_lock` on supported platforms, with pthread mutex as the
 // fallback.
 internal class Lock {
+	// Both `UnfairLock` and `PthreadLock` use `ManagedBufferPointer` to allocate inline
```

Comment:
One less level of pointer chasing at runtime. Since the header is part of the class allocation `ManagedBufferPointer`, it can be resolved at static time as an offset to `self`.